pymmcore/__init__.pyi

```diff
@@ -1014,10 +1014,22 @@ class CMMCore:
         """Sets the current slm device."""
     def setSLMExposure(self, slmLabel: str, exposure_ms: float) -> None:
         """For SLM devices with build-in light source (such as projectors),
-
         this will set the exposure time, but not (yet) start the illumination"""
+    @overload
+    def setSLMImage(self, slmLabel: str, pixels: np.ndarray[np.uint8]) -> None:
```

Comment:
```suggestion
    def setSLMImage(self, slmLabel: str, pixels: np.ndarray[Any, np.dtype[np.uint8]]) -> None:
```

I believe this is how this needs to look if you want to use `np.ndarray[]` as a generic directly (it's a two argument generic).  Slightly nicer one-arg syntax is by importing `numpy.typing`:

```python
import numpy.typing as npt

# then use:
def setSLMImage(slmLabel: str, pixels: npt.NDArray[np.uint8]) -> None:
```

see here
https://numpy.org/devdocs/reference/typing.html#numpy.typing.NDArray